install/kubernetes/tetragon/templates/operator_deployment.yaml

```diff
@@ -49,7 +49,7 @@ spec:
           {{- with .Values.tetragonOperator.extraVolumeMounts }}
             {{- toYaml . | nindent 10 }}
           {{- end }}
-        {{- with .Values.tetragonOperator.securityContext }}
+         {{- with .Values.tetragonOperator.containerSecurityContext}}
```

Comment:
you have a space here, not sure it's intended. Anyway should be change for backward compatibility on previous field name.